### PR TITLE
Classes and objects tests

### PR DESCRIPTION
**Pull request by Kleo5522**
Classes and Objects homework.

### CODE REVIEW

File: ClassesAndObjectsTestsTests/Exercise1Tests.cs [hunk 1]
```diff
@@ -1,20 +1,38 @@
...
+
+            //Act
+            var result = exercise1.PrintProduct();
+
+            //Assert
+            Assert.AreEqual(expected,result);
+        }
+
+        [TestMethod]
+        public void Exercise1Test_ChangePrice20Point47AndQuantity10_ReturnStringNewPrice20Point47And10Quantity()
```
Comment (ausmoons) [inline ClassesAndObjectsTestsTests/Exercise1Tests.cs:24]: PrintProduct_ChangePrice20Point47AndQuantity10_ReturnStringNewPrice20Point47And10Quantity

**Review by ausmoons — CHANGES_REQUESTED** [review]
Izlabo namingu Metode_KoPadod_KoSagaida

File: ClassesAndObjectsTestsTests/Exercise12Tests.cs [hunk 2]
```diff
@@ -21,46 +21,46 @@ public void Exercise12Test_CreateNewAccountNameJohnAmount120_ExpectedMessageJohn
         }
 
         [TestMethod]
-        public void Exercise12Test_CreateNewAccountNameJohnAmount120Deposit50Point99_ExpectedMessageJohnAnd170Point99()
+        public void Deposit_Balance120Deposit50Point99_ReturnsBalance170Point99()
```
Comment (ausmoons) [inline ClassesAndObjectsTestsTests/Exercise12Tests.cs:24]: Kādu metodi tu te testē?

Comment (Kleo5522) [reply ClassesAndObjectsTestsTests/Exercise12Tests.cs:24]: Deposit Metodi.

**Review by ausmoons — APPROVED** [review]

**Pull request merged**